### ISSUE TITLE
[Accessibility][Name, Role, Value - Create Node]: Role of Paste button is not correct in add node section.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2076]
**User Experience** 
Users who rely on AT will might get confused if the controls available in the same menu item have different roles.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and set the zoom at 400%.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the flow chart.
4. Navigate to "+" button and activate it.
5. Navigate to 'Paste' available in the list and observe the issue.

**Actual:** 
The narrator narrate the role of 'Paste' as button where as all the other option's are having role of menu item. The narrator narrates 2nd element i.e. 'Send a respond' as menu item 2 of 8.

**Expected:**
The role of 'Paste' should be same as other's. The role of 'Paste' should be menu item and narrator should narrates as 'Paste' menu item 1 of 8.

**MAS Impacted:** [MAS4.1.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment:**
[MAS4.1.2_Role of paste button.zip](https://github.com/microsoft/BotFramework-Composer/files/4249382/MAS4.1.2_Role.of.paste.button.zip)

**Comment by corinagum** [comment]
![MAS4 1 2_role of paste button](https://user-images.githubusercontent.com/14900841/75415293-136c3a80-58e0-11ea-8359-0b8c86f96f66.png)

.png uploaded for ease of access, but note that there is also a video in the .zip

**Comment by cwhitten** [comment]
addressed in https://github.com/microsoft/BotFramework-Composer/pull/2126

**Comment by ashish315** [comment]
As cross-checked the issue is fixed in master version so we are good to close the issue.
